src/main/scala/io/ino/solrs/AsyncSolrClient.scala

```diff
@@ -243,7 +243,7 @@ class AsyncSolrClient private (val loadBalancer: LoadBalancer,
     val promise = scala.concurrent.promise[QueryResponse]
     val startTime = System.currentTimeMillis()
 
-    val url = solrServer.baseUrl + getPath(q) + ClientUtils.toQueryString(wparams, false)
+    val url = solrServer.baseUrl + getPath(q) + wparams.toQueryString
```

Comment:
I'd prefer to keep support for solr 5.0, therefore we should use ClientUtils.toQueryString if available and otherwise wparams.toQueryString (both via reflection).